src/Mod/Draft/draftguitools/gui_snapper.py

```diff
@@ -191,14 +236,12 @@ def snap(self, screenpos,
 
         self.running = True
 
-        global Part, DraftGeomUtils
-        import Part, DraftGeomUtils
```

Comment:
The reason the modules are placed in this function is because we don't want to globaly import `Part` when we import the snapper. According to Yorik this makes loading of Draft quite slow, so for the time being, I would leave these imports here. Maybe I'd remove the `global` keyword.

To be honest I didn't check this thoroughly because I was doing many things. But eventually I need to test this better.